internal/ibazel/ibazel.go

```diff
@@ -532,21 +532,29 @@ func (i *IBazel) watchFiles(query string, watcher common.Watcher) {
 		return
 	}
 
+	rootParent, _ := filepath.Split(toWatch[0])
+	rootSplit := strings.Split(strings.Trim(rootParent, "/"), "/")
```

Comment:
https://pkg.go.dev/path/filepath#SplitList
```suggestion
	rootSplit := filepath.SplitList(rootParent)
```

Reply:
I started with split list as well, but it actually does something different. From the example:

```
fmt.Println("On Unix:", filepath.SplitList("/a/b/c:/usr/bin"))
```

```
On Unix: [/a/b/c /usr/bin]
```

Reply:
Oh yeah you're right, the docs do say that. I guess `SplitList` means `Split applied to path lists`, not `Split returning a list`.